src/Shout/SHRBTextStyler.class.st

```diff
@@ -1101,10 +1101,10 @@ SHRBTextStyler >> visitSequenceNode: aSequenceNode [
 
 { #category : #visiting }
 SHRBTextStyler >> visitVariableNode: aVariableNode [
-	self
-		addStyle: (self resolveStyleFor: aVariableNode)
-		attributes: (self resolveVariableAttributesFor: aVariableNode)
-		forNode: aVariableNode
+
+	self addStyle: (self resolveStyleFor: aVariableNode) attributes: (self resolveVariableAttributesFor: aVariableNode) forNode: aVariableNode.
+
+	self visitMethodComments: aVariableNode comments
```

Comment:
Visiting comments is not automatic? Because comments can be attached to any nodes.

Reply:
From what I remember, the AST visitor does not visit comments yet.

Comments are not yet well integrated in the AST. There have been improvements during the past few years but there are still work to do.
Visiting comments is something that we need to do. But as always, improvements require time. Cf. https://github.com/pharo-project/pharo/issues/12942

Reply:
Hum. Since adding explicit comment visit on each node is insane and having a better visitor takes work and time. A global workaround can be to perform a second quick pass only to highlight comments (that override the previous style).